redbot/core/utils/antispam.py

```diff
@@ -45,14 +45,13 @@ def __interval_check(self, interval: AntiSpamInterval):
     @property
     def spammy(self):
         """
-        use this to check if any interval criteria are met
+        Check whether any intervals are active.
         """
         return any(self.__interval_check(x) for x in self.__intervals)
 
     def stamp(self):
         """
-        Use this to mark an event that counts against the intervals
-        as happening now
+        Mark an event against the intervals as happening now.
```

Comment:
I don't really know how to describe this function better but I do think it's not easy to figure out what it means if you don't already understand what this class does.